src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -76,6 +77,31 @@ inline bool BinaryBroadcastShape(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+inline bool BinaryBroadcastStorageTypeCsr(const nnvm::NodeAttrs& attrs,
```

Comment:
No need to put Csr in the title since we may add row_sparse in the same function in future. Also the name is confusing because this is only for mul/div

Reply:
Changed to BinaryBroadcastMulStorageType